frontend/web/micro-ui-internals/packages/modules/PublicServices/src/pages/employee/DigitDemo/modulePageComponent.js

```
@@ -0,0 +1,62 @@
+import React from "react";
+import { Card, Button, HeaderComponent, CardText, Loader, SubmitBar } from "@egovernments/digit-ui-components";
+import { useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+import { transformResponseforModulePage } from "../../../utils";
+import { Link } from "react-router-dom/cjs/react-router-dom.min";
+
+const modulePageComponent = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)